### ISSUE TITLE
Clarification needed

### ISSUE THREAD

**Issue by scottaohara** [ThePacielloGroup/WAI-ARIA-Usage#19]
Per this [aria-readonly on an element with contenteditable test](https://w3c.github.io/html-aria/tests/contenteditable-readonly-test.html), the ARIA validator is correctly indicating that `aria-readonly` is invalid, but not for the expected reason.

There should be a rule that explicitly looks for if an element has `contenteditable`, and if so, throw an error if `aria-readonly=true` is also set to that element.

**Comment by gezlemon** [comment]
Added contenteditable rule with aria-readonly
